README.md

```diff
@@ -30,7 +30,7 @@ Feel free to submit a [pull request](https://github.com/EddieHubCommunity/awesom
 
 2. Fork the project, using the top right corner button, that states `fork`
 
-3. Create a file named `your-username.json` in the directory [`profiles`](/profiles) with the following content.
+3. Create a file named `your-username.json` in the directory [`profiles`](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) with the following content.
```

Comment:
Should we use the link to the profiles folder, or to the [profiles page](https://eddiehubcommunity.github.io/awesome-github-profiles/profiles)?

Reply:
https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles

Guess this link is fine

Reply:
I think profile folder would be the choice, because we are letting the people know where they can add their profile.